Lib/test/test_typing.py

```diff
@@ -7223,10 +7223,29 @@ class Wrong(*bases):
                         pass
 
     def test_is_typeddict(self):
-        assert is_typeddict(Point2D) is True
-        assert is_typeddict(Union[str, int]) is False
+        self.assertTrue(is_typeddict(Point2D))
+        self.assertFalse(is_typeddict(Union[str, int]))
```

Comment:
+1 for using the unittest methods, but note that this isn't a _direct_ translation. `assertTrue` only asserts that the thing is truthy, not that it actually is the `True` constant.

Reply:
Oh good point, I'll switch to `assertIs(..., True)`.